kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -1,19 +1,19 @@
 package com.wepay.kafka.connect.bigquery;
 
-import com.google.cloud.bigquery.BigQuery;
-import com.google.cloud.bigquery.StandardTableDefinition;
-import com.google.cloud.bigquery.TableId;
-import com.google.cloud.bigquery.TableInfo;
-import com.google.cloud.bigquery.TimePartitioning;
 
+import com.google.cloud.bigquery.*;
```

Comment:
Nit: do we want to separate `com.google.cloud.bigquery` from `com.wepay.kafka.connect.bigquery`? And do we want to import `.*` instead of specific modules?

Reply:
I tried to import classes separately but IntelliJ forced them to be `import .*`

Reply:
In general, prefer explicit import. IntelliJ can be configured to disable this `*` feature.

Reply:
Got it! Updated accordingly.